packages/guideline-blocks-settings/src/components/Link/LinkInput.tsx

```diff
@@ -0,0 +1,79 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AppBridgeBlock } from '@frontify/app-bridge';
+import { ButtonSize, Checkbox, CheckboxState, FormControl, TextInput, TooltipPosition } from '@frontify/fondue';
+import { LinkSelector } from './LinkSelector';
+
+type LinkInputProps = {
+    url?: string;
+    info?: string;
+    label?: string;
+    required?: boolean;
+    newTab?: CheckboxState;
+    openInNewTab?: boolean;
+    onUrlChange?: (value: string) => void;
+    onToggleTab?: (checked: boolean) => void;
+    isValidUrlOrEmpty?: (url: string) => boolean | undefined;
+    appBridge: AppBridgeBlock;
+    clearable?: boolean;
+    placeholder?: string;
+    buttonSize?: ButtonSize;
+    hideInternalLinkButton?: boolean;
+};
+
+export const LinkInput = ({
+    onUrlChange,
+    onToggleTab,
+    isValidUrlOrEmpty,
+    appBridge,
+    clearable,
+    placeholder,
+    newTab,
+    openInNewTab,
+    url = '',
+    required,
+    info,
+    label,
+    buttonSize,
+    hideInternalLinkButton,
+}: LinkInputProps) => {
+    const isUrlValid = isValidUrlOrEmpty ? isValidUrlOrEmpty(url) : 'import';
+    const checkedState = newTab ?? (openInNewTab ? CheckboxState.Checked : CheckboxState.Unchecked);
+    return (
+        <div data-test-id="link-input">
+            <FormControl
+                label={{
+                    children: label,
+                    htmlFor: 'url',
+                    required,
+                    tooltip: info ? { content: info, position: TooltipPosition.Top } : undefined,
+                }}
+            >
+                <TextInput
+                    id="url"
+                    value={url}
+                    clearable={clearable}
+                    onChange={onUrlChange}
+                    placeholder={placeholder ?? 'https://example.com'}
+                    focusOnMount
+                />
+            </FormControl>
+            {!isUrlValid && <div className="tw-text-red-65 tw-mt-1 tw-text-s">Please enter a valid URL.</div>}
```

Comment:
`tw-text-red-65` izn't a valid token

Reply:
![image](https://github.com/Frontify/brand-sdk/assets/30796791/985da7fb-324b-48fe-8673-fd070c24f74f)
![image](https://github.com/Frontify/brand-sdk/assets/30796791/ffc10dc0-023e-4f07-b6d3-8b476e76add8)
 
 It seems like it is.

Reply:
Sorry I meant part of these tokens: https://beta-fondue-components.frontify.com/?path=/story/tokens--alias-tokens

Reply:
Ah ok, fixed